copilot/fsd-assessment-store/manifest.yml

```diff
@@ -47,12 +47,18 @@ variables:
   FLASK_ENV: ${COPILOT_ENVIRONMENT_NAME}
   APPLICATION_STORE_API_HOST: http://fsd-application-store:8080
   ACCOUNT_STORE_API_HOST: http://fsd-account-store:8080
+  FUND_STORE_API_HOST: http://fsd-fund-store:8080
+  ASSESSMENT_API_HOST: "https://assessment.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk"
```

Comment:
While looking through this I was slightly mislead by `ASSESSMENT_API_HOST` at a glance assuming that meant the assessment API (which my brain replaced with store)

I wonder if this could be clarified with something like `ASSESSMENT_FRONTEND_HOST` or `CURRENT_DOMAIN_HOST` (as its for the app being deployed?)